### DISCUSSION TITLE
DCC setting with slider

### DISCUSSION THREAD

**Discussion by MilackB** [jilleb/MQB-FPA#49] [Q&A]
Hi, jilleb.  I watched your post. And I'd like to get this mod
 https://golfmk7.com/forums/index.php?threads/dcc-suspension-mod-by-installing-the-dsc-sport-controller.385091/post-7653393 
I ride MY2020 GOLFMK7.5 GTI TCR. 
Gateway module : SW number 3Q0907530AC HW number 3Q0907503C
I have VCP,VCDS and OBDeleven. But I cannot access ODIS online.

My question is
1:Is it able to flash my Gateway module with New FW of Arteon ?
2:How can I get XML file with FPA setting？
3:Which parameter should I change for enabling slider control?

Sorry for the rudimentary question.
Regards.
![PaAAAgMIb-A-960](https://user-images.githubusercontent.com/96638842/147375731-4ab15e95-68c6-4e80-99b1-318f5a4e4738.jpg)

**Comment by olli991** [comment]
1: You can flash it possible but the Arteon DS is not that good on hatchbacks because the wheelbase is shorter than on the Arteon. It fits better on estates.
2: You can grab you own DS via ODIS Online with your VIN or just use one what comes close to your car
3: Compare your DS with this from the excamples ([HERE](https://github.com/jilleb/MQB-FPA/blob/7afc35e16b291dcf46c939bccf61209b2a1d0acd/Full%20Example%20for%20DCC%20Index%20C%20with%20slider)) and implement it into your DS
